CollectionViewShelfLayout/CollectionViewShelfLayout.swift

```diff
@@ -381,6 +392,9 @@ private class TrackingScrollView: UIScrollView {
     didSet {
       trackingView?.addGestureRecognizer(panGestureRecognizer)
       frame = trackingView?.bounds ?? .zero
+      translatesAutoresizingMaskIntoConstraints = false
+      trackingView?.insertSubview(self, at: 0)
+      alpha = 0
```

Comment:
I think we can move `alpha = 0` into its initializers and also set it `isHidden` property to false